lib/index.test.js

```diff
@@ -134,6 +134,29 @@ describe('getEntity', () => {
   })
 })
 
+describe('getProduct', () => {
+  it('works on basic url', () => {
+    expect(getProduct('https://www.youtube.com/embed/alGcULGtiv8')).toMatchInlineSnapshot(`
```

Comment:
we don't have any statistics here that need updating so let's use `toEqual` instead of a snapshot :)